chain/hedera/tx_input/tx_input.go

```diff
@@ -0,0 +1,100 @@
+package tx_input
+
+import (
+	"time"
+
+	xc "github.com/cordialsys/crosschain"
+	"github.com/cordialsys/crosschain/factory/drivers/registry"
+)
+
+// TxInput for Hedera
+type TxInput struct {
+	xc.TxInputEnvelope
+	// Sender account id in `0.0.12345` format
+	AccountId string `json:"account_id"`
+	// Node account id in `0.0.12345` format
+	NodeAccountID string `json:"node_account_id"`
+	// Valid transaction timestamp in unix nanos
+	ValidStartTimestamp int64 `json:"valid_start_timestamp"`
+	// Max fee that the transaction can consume
+	MaxTransactionFee uint64 `json:"max_transaction_fee"`
+	// Transaction is considered outdated after: `ValidStartTimestamp + ValidTime`
+	// Max 180 seconds
+	ValidTime int64 `json:"valid_time"`
+	// Transaction memo, max 100 characters
+	Memo string `json:"memo"`
+}
+
+var _ xc.TxInput = &TxInput{}
+var _ ValidStartGetter = &TxInput{}
+
+type ValidStartGetter interface {
+	GetTimestampNano() int64
+	GetExpiration() int64
+}
```

Comment:
This may be more usable.

```go
type ValidStartGetter interface {
	GetTimestampNano() int64

	GetTimestamp() time.Time
	GetExpiration() time.Time
}
```